Resources/views/js.html.twig

```diff
@@ -23,6 +23,13 @@
             this.language = '{{ language }}';
         {%- endif -%}
 
+        {%- if callback is defined -%}
+            var fn = window['{{callback}}'];
```

Comment:
This seems a bit hacky to me, but the more I think about it, I think it's an easy way to implement this and would be easy to use :). What do you think about renaming to `newCommentCallbackFunctionName`?